app/models/concerns/dossier_rebase_concern.rb

```
@@ -58,6 +58,7 @@ def rebase
       .joins(:type_de_champ)
       .group_by(&:stable_id)
       .transform_values { Champ.where(id: _1) }
+      .tap { _1.default = Champ.none }
```

ce `_1.default` est obscure pour moi.

---

ça veut dire que `hash[:unknown]` return `Champ.none`

---

on le fait déjà plus haut sur un autre `group_by`